CONTRIBUTING.md

```diff
@@ -206,6 +206,18 @@ Alternative manual steps for IntelliJ.
    3. Navigate to the file `build-conventions/formatterConfig.xml`
    4. Click "OK"
 
+#### Options
+
+When importing to IntelliJ, we offer a few options that can be used to
+configure the behaviour of the import:
+
+| Property                                   | Description                                                                                          | Values (* = default) |
+|--------------------------------------------|------------------------------------------------------------------------------------------------------|----------------------|
+| `org.elasticsearch.idea-configuration-cache` | Should IntelliJ enable the Gradle Configuration cache to speed up builds when generating run configs | *`ture`, `false`         |
+| `org.elasticsearch.idea-delegate-to-gradle`  | Should IntelliJ use Gradle for all generated run / test configs or prompt each time                  | `true`, *`false`         |
+
+These options can be set anywhere on the Gradle config path including in `~/.gradle/gradle.settings`
```

Comment:
We should just use `~/.gradle/gradle.properties` for this as there's build-in support to read props in gradle from that file. I'm surprised `gradle.settings` even works

Reply:
Nope not a typo. Stuff in `~/.gradle/gradle.settings` applies globally for the current user but the repo specific file would work too. I guess it comes down to if we think it would be useful for it to apply to multiple projects or not (IE would I like to to just globally apply to stateful and serverless assuming they share this build code?).

I could be convinced either way or, perhaps we just document the options here and leave it up to the user if they want it applied globally or just per repo?

Reply:
Just realised this is totally a typo -.- Updating now